packages/bus-query-operation/lib/ActorQueryOperation.ts

```diff
@@ -249,6 +249,10 @@ export interface IActorQueryOperationOutputBindings extends IActorQueryOperation
    * The stream of bindings resulting from the given operation.
    */
   bindingsStream: BindingsStream;
+  /**
```

Comment:
These new fields being optional makes them very inconvenient to use.

In the actor-init-sparql package, I would suggest creating the following instead:
* IQueryResultBindings: extends from IActorQueryOperationOutputBindings, and adds this new bindings method.
* IQueryResultQuads: extends from IActorQueryOperationOutputQuads, and adds this new quads method.
* IQueryResultBoolean: extends from IActorQueryOperationOutputBoolean, and adds nothing.
* IQueryResult: _union type_ over the three above.

The return type of `query` in ActorInitSparql can then be changed to return `IQueryResult`.
Then the "optional" any "any" stuff won't be needed anymore.